packages/lesswrong/server/migrations/20221220T125246.fix_index_uniqueness.ts

```diff
@@ -0,0 +1,19 @@
+import { getAllCollections } from "../../lib/vulcan-lib/getCollection";
+import { createIndex, dropIndex } from "./meta/utils";
+
+export const up = async ({db}: MigrationContext) => {
+  for (const collection of getAllCollections()) {
+    if (collection.isPostgres()) {
+      const table = collection.getTable();
+      for (const index of table.getIndexes()) {
+        if (index.isUnique() || index.getPartialFilterExpression()) {
+          try {
+            await dropIndex(db, collection, index);
```

Comment:
In principle it could be a bit dodgy, but in practice the only indexes that this affects are so small that they can be droped and recreated in well under a second so it's basically fine (and doing it "properly" is a real PITA).